cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/CruiseControl/RebalanceOptions.java

```diff
@@ -0,0 +1,76 @@
+package io.strimzi.operator.cluster.operator.assembly.CruiseControl;
+
+import java.util.List;
+
+public class RebalanceOptions {
+
+    private boolean isDryRun;
+    private boolean replyWithJson;
+    private List<String> goals;
+
+    public boolean isDryRun() {
+        return isDryRun;
+    }
+
+    public boolean willReplyWithJson() {
```

Comment:
what do you think about leaving this out? I think that we should always work in JSON format so not providing a way to allow a different response format.

Reply:
True, I will just assume it is always true